test/executor.test.js

```diff
@@ -53,11 +53,17 @@ describe('executor', function() {
   });
 
   describe('when booting', function() {
-    it('should set the booting status', function(done) {
+    it('should set the `booting` flag during execution', function(done) {
       expect(app.booting).to.be.undefined();
-      boot.execute(app, dummyInstructions, function(err) {
+
+      boot.execute(app, simpleAppInstructions(), function(err) {
         expect(err).to.be.undefined();
+
+        var bootingFlagSet = process.bootFlags.indexOf('booting') > -1;
+        expect(bootingFlagSet).to.be.ok();
```

Comment:
Please consider learning yourself more familiar with the different assertions provided by Chai, see http://chaijs.com/api/bdd/. The benefit of these assertions (compared to your solution above) is a much more helpful error message reported when the check fails.